src/main/kotlin/exercise3/task2/ParenthesesClusters.kt

```diff
@@ -24,9 +24,29 @@ package exercise3.task2
  */
 
 internal fun String.splitToBracketsClusters(): List<String> {
-    TODO("Implement me!!!")
-}
+    val open = listOf('(', '[', '{')
```

Comment:
The solution gives a wrong result for a case when the receiver string is empty. 
Instead of returning the `[]` empty list, it returns the list with the empty string `[""]`.

Other than this, the solution is well-written and easy to read and understand.

Reply:
Oh, so that was why it was failing xD

Tester was reporting that it was expecting [] but actual value was [], which left me pretty confused since it printed the same thing, so I just ignored it. 
I really didn't notice that it was actually an empty string haha